src/transformers/models/bloom/tokenization_bloom_fast.py

```diff
@@ -109,7 +99,7 @@ class BloomTokenizerFast(PreTrainedTokenizerFast):
 
     vocab_files_names = VOCAB_FILES_NAMES
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
-    max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+    max_model_input_sizes = []
```

Comment:
I am really in favor of this change, especially because with AliBi positional embeddings we don't have a max model input size (or pretrainned positional embeddings)

Reply:
Your proposition looks good to me! :smile: